skyVegasTests/utilities/validateConsoleMessagesHelper.ts

```diff
@@ -4,17 +4,26 @@ import { messageExists } from '../utilities';
 export const validateConsoleMessages = async (
     expectedMessage: string,
     consoleMessages: string[],
-    intervals: number[] = [5_000, 10_000, 15_000],
-    timeout: number = 60_000
+    deleteMessages = false,
+    intervals: number[] = [5_000, 10_000, 15_000, 20_000],
+    timeout = 60_000
```

Comment:
minor suggestion which might add some code clarity, maybe having a const for second i.e. 1000 and whenever we need a time, calculate it in terms of that e.g.
timeout = 60 * second
intervals = [5 * second, 10 * second, ...]
although I appreciate we use these timings in a lot of places so would be many of changes for not much benefit

Reply:
on second thought, this is a waste of time as we have plenty of millisecond timings scattered around as well